src/test/java/co/rsk/bitcoinj/script/NoRedeemScriptParserTest.java

```diff
@@ -0,0 +1,15 @@
+package co.rsk.bitcoinj.script;
+
+import co.rsk.bitcoinj.core.ScriptException;
+import org.junit.Test;
+
+public class NoRedeemScriptParserTest {
+
+    private final RedeemScriptParser noRedeemScriptParser = new NoRedeemScriptParser();
+
+    @Test (expected = ScriptException.class)
+    public void extractStandardRedeemScript_whenNoRedeemScriptIsCalled_shouldThrowScriptException() {
```

Comment:
i think theres no need to say `whenNoRedeemScriptIsCalled`, its implied since you are testing the `NoRedeemScriptParser`

Reply:
Done! Now the test name is `extractStandardRedeemScript_shouldThrowScriptException`.